### ISSUE TITLE
[Feature Request] Add support for bottle custom paths for CrossOver

### ISSUE THREAD

**Issue by mavethee** [Heroic-Games-Launcher/HeroicGamesLauncher#3815]
### Problem description

With any other method you can define Wine prefix / bottle path but as long as selecting CrossOver I'm forced to default path which makes games unbootable as they can't see bottle in different path.

### Feature description

Add possibility to change bottle paths, would help for running games from Heroic in modified CrossOver instances like the CXPatcher modifications that can boost perfs in some games.

**Comment by Kief5555** [comment]
Yes please, this would help improve stability to the games especially because CXPatcher does exactly that.

**Comment by macinlew** [comment]
With 2.16.0 if I set the Bottle Folder in CrossOvers (25B3) Settings, System Integration to default, it works and continues to work when I set the Bottle Folder to my external volume. I just have to use the default to launch one game, then setting the Bottle Folder to my external volume works from then on.
